lib/rules/no-unnecessary-act.ts

```diff
@@ -89,7 +124,16 @@ export default createTestingLibraryRule<[], MessageIds>({
           node: identifierNode,
           messageId: 'noUnnecessaryActEmptyFunction',
         });
-      } else if (!hasSomeNonTestingLibraryCall(blockStatementNode.body)) {
+        return;
+      }
+
+      const isReported =
```

Comment:
I found this var name confusing! I thought it meant the code violation was already reported, but it actually indicates if it needs to be reported. Could you rename it to something like `shouldBeReported`?